Dockerfile

```diff
@@ -14,9 +14,9 @@ ENV GROUP geodata
 RUN apt-get update -qq \
     && DEBIAN_FRONTEND=noninteractive apt-get install -qq -y --upgrade ca-certificates \
     && DEBIAN_FRONTEND=noninteractive apt-get install -qq -y  \
-        libgeos-dev \
-        gettext-base \
-        apache2 libapache2-mod-wsgi-py3 \
+    libgeos-dev \
```

Comment:
Oops some unwanted whitespaces changes, probably done via vscode, see https://github.com/geoadmin/tool-dev-setup/pull/114

Reply:
as the variable is called app_version all over, I would like to keep it app_version